packages/@aws-cdk/aws-route53/lib/util.ts

```diff
@@ -49,7 +49,7 @@ class ValidationError extends Error {
  *        <li>Otherwise, append +.+, +zoneName+ and a trailing +.+</li>
  *      </ul>
  */
-export function determineFullyQualifiedDomainName(providedName: string, hostedZone: IHostedZone): string {
+export function determineFullyQualifiedDomainName(providedName: string, hostedZone: IHostedZone, forceRemovePeriodProp?: boolean): string {
```

Comment:
I'm not a fan of the name `forceRemovePeriodProp` - I used that as an example. `forceRemovePeriod` is fine, but honestly I'm not great at coming up with names. If you have any other suggestions, feel free. Something descriptive like `removeDomainNamePeriod` is much better in my opinion.